api.yaml

```diff
@@ -226,6 +246,21 @@ definitions:
     required:
       - id
       - content
+  Device:
+    title: Device
+    description: Device data.
+    type: object
+    properties:
+      installationId:
+        type: string
+      platform:
+        type: string
+      pushChannel:
+        type: string
```

Comment:
and this one as well?

Reply:
`platform` of course (in the code is an enum). pushChannel is the PNS token of the device